src/arena.c

```diff
@@ -1061,8 +1160,14 @@ arena_slab_alloc(tsdn_t *tsdn, arena_t *arena, szind_t binind,
 	extent_hooks_t *extent_hooks = EXTENT_HOOKS_INITIALIZER;
 	bool zero = false;
 	bool commit = true;
-	extent_t *slab = extent_alloc_cache(tsdn, arena, &extent_hooks, NULL,
-	    bin_info->slab_size, 0, PAGE, &zero, &commit, true);
+	extent_t *slab = extents_alloc(tsdn, arena, &extent_hooks,
+	    &arena->extents_dirty, NULL, bin_info->slab_size, 0, PAGE, &zero,
+	    &commit, true);
+	if (slab == NULL) {
+		slab = extents_alloc(tsdn, arena, &extent_hooks,
+		    &arena->extents_muzzy, NULL, bin_info->slab_size, 0, PAGE,
```

Comment:
We should probably document that alloc from muzzy should be the reverse order of kernel reclaiming (LRU), so that we have a higher chance of the pages not being unmapped yet.